UnityProject/Assets/Scripts/Machines/Microwave.cs

```diff
@@ -15,9 +15,10 @@ public class Microwave : NetworkBehaviour
 
 	/// <summary>
 	/// Time left until the meal has finished cooking (in seconds).
+	/// I don't see the need to make it a SyncVar.  It will economize network packets.
 	/// </summary>
 	[HideInInspector]
-	public float microwaveTimer = 0;
+	public float MicrowaveTimer = 0;
```

Comment:
Fields should be camelCase. Use PascalCase for properties. Not a big issue so will merge this.